x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_logic.test.ts

```diff
@@ -27,7 +27,7 @@ describe('CredentialsLogic', () => {
       write: true,
       access_all_engines: true,
     },
-    activeApiTokenIsExisting: false,
+    isActiveApiTokenExisting: false,
```

Comment:
This still reads a tiny bit awkwardly to me - what are your thoughts on `hasActiveApiToken` instead? 🤔 

Or if you're not a fan, we could forego the is/has/should boolean prefixes for this file and stick with `activeApiTokenExists` (still reads a lot better than `IsExisting` just IMO, and reuses the `activeApiToken*` naming we have going on in a few values)

Reply:
There's always an activeApiToken, it's just a matter of whether it's an existing token that we're editing, or one that we're creating, so `hasActiveApiToken` won't work.

`activeApiTokenExists` would work, or `doesActiveApiTokenExist`.

I'll go with `activeApiTokenExists`.